planner/core/plan.go

```diff
@@ -380,6 +380,9 @@ type PhysicalPlan interface {
 	// Clone clones this physical plan.
 	Clone() (PhysicalPlan, error)
 
+	// return cost of this physical plan.
+	Cost() float64
```

Comment:
This interface seems duplicated, better to use `core.GetPlanCost(plan, property.RootTaskType)` directly.